shaptools/hana.py

```diff
@@ -44,6 +48,7 @@ class HanaInstance:
     """
 
     PATH = '/usr/sap/{sid}/HDB{inst}/'
+    INSTALL_EXEC = '{software_path}/DATA_UNITS/HDB_LCM_LINUX_X86_64/hdblcm'
```

Comment:
AFAIK, this path changes accordingly with the Architecture. I think would make sense to check the current architecture and compose this for x86_64 or ppc64le.

Reply:
@diegoakechi do you know if these two are the unique valid options? Otherwise we can let the user the set the whole path, this would be a safer option

Reply:
@arbulu89 These are the only two supported SAP Architectures at this moment. Even that more appear, I think we should handle this internally on the Library, as it is complicated to the user to inform this value. Only to get this,  he needs to unpack the downloaded .SAR and dig into the folders to find the Life Cycle Manager binaries location.

Reply:
New ticket created to fix this point:
https://trello.com/c/QFlTJfWU/313-add-dynamic-path-search-for-hdbclm-command